mesonbuild/compilers/compilers.py

```diff
@@ -334,6 +334,17 @@ def get_option_value(options: 'KeyedOptionDictType', opt: OptionKey, fallback: '
     return v
 
 
+def disable_debug(options: KeyedOptionDictType) -> bool:
```

Comment:
Wouldn't `is_debug_disabled` or something similar be a more appropriate name, since this function do not *disable* anything...

Reply:
yeah, that makes sense.

Reply:
I went with `are_assertions_disabled`, since this is really checking for assertions, and not debug builds.